tests/testthat/test-profile_emissions.R

```diff
@@ -179,31 +181,6 @@ test_that("yields a single distinct value of `*matching_certainty_company_averag
   expect_equal(unique(count$n_distinct_matching_certainity_per_company), 1.0)
 })
 
-test_that("total number of rows for a comapny is either 1 or 4", {
-  skip_unless_tilt_indicator_is_newer_than("0.0.0.9206")
```

Comment:
SELF: Study the case and share a reprex before and after to show and record the new data structure.
I believe the number of rows per benchmark is covered in tests of summarize_co2_range(). Confirm.